### PR TITLE
Update netservices.md and installation-guide

### PR DESCRIPTION
**Pull request by Coldfirex**
Add telnetd section, update wording, add clarifications, and switch steps to GUI as the services config file does not contain those items anyways unless you manually enable the services via the GUI.

### CODE REVIEW

File: content/guides/daily-tasks/netservices.md [hunk 1]
```diff
@@ -18,17 +18,19 @@ repeat new password:</pre></p>
...
 You can change the hostname by editing /system/settings/network/hostname.</div>
+<br>
 
-<h3>ftpd - an FTP server daemon</h3>
+<h3>ftpd - FTP server daemon</h3>
 Haiku can easily be made to listen for incoming FTP connections to serve its files.
 
-<h4>Start ftpd at boot</h4>
-<p>Haiku will automatically start its FTP server daemon at boot if you remove the comment hashes (#) from the service ftp section of /boot/system/settings/network/services.
+<p>To start the FTP server select the Deskbar (the blue feather icon), Preferences, Network.  Select FTP Server and enable.</p>
```
Comment (humdingerb) [inline content/guides/daily-tasks/netservices.md:26]: It's actually a blue leaf. :)
IMO we can assume the users know there way around Haiku's most basic usage, i.e. the Deskbar and how to launch apps and preferences with it.

```suggestion
<p>To start the FTP server open the Network preferences, select **`FTP server`**  from its list of services and click **`Enable`**.</p>
```

The same goes for the telnet server below.

**Review by humdingerb — COMMENTED** [review]
Doesn't the starting of the SSH server work also by enabling it from the Network preference panel? If so, it could be added as an alternative to launching from Terminal via /boot/system/bin/sshd

**Comment by Coldfirex** [comment]
> 
> 
> Doesn't the starting of the SSH server work also by enabling it from the Network preference panel? If so, it could be added as an alternative to launching from Terminal via /boot/system/bin/sshd

Its actually already started by default.  Updated verbage under ssh as well now.

File: content/guides/daily-tasks/netservices.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ date = "2010-05-11T15:29:32.000Z"
 tags = []
 +++
 
-The basic installation of Haiku contains several standard system network utilities including SSH and Telnet
+The basic installation of Haiku contains several standard system network services including a FTP, Telnet, and SSH Server.
```
Comment (humdingerb) [inline content/guides/daily-tasks/netservices.md:8]:
```suggestion
The basic installation of Haiku contains several standard system network services including an FTP, Telnet, and SSH server.
```

File: content/guides/daily-tasks/netservices.md [hunk 3]
```diff
@@ -18,29 +18,34 @@ repeat new password:</pre></p>
...
-<h3>sshd - the secure shell daemon</h3>
+DISCLAIMER: telnet traffic is unencrypted.
+
+<p>To start the Telnet server open the Network preferences, select **`Telnet server`**  from its list of services and click **`Enable`**.</p>
+<br>
+<h3>sshd - Secure shell daemon</h3>
 SSH is a network protocol that allows for data to be exchanged using a secure channel between two networked devices. The most common use is shell access.
 
-Starting the SSH server daemon in Haiku is easy:
+The SSH server daemon in Haiku is already started on a deafult install, but it does not allow connections until configured.
```
Comment (humdingerb) [inline content/guides/daily-tasks/netservices.md:40]:
```suggestion
The SSH server daemon is set to start automatically on a default Haiku install, but it does not allow connections until configured.
```

File: content/guides/daily-tasks/netservices.md (hunk 2 above)
Comment (Coldfirex) [reply content/guides/daily-tasks/netservices.md:8]: "a FTP", otherwise looks great.

Comment (Coldfirex) [reply content/guides/daily-tasks/netservices.md]: reuploaded revision

**Review by Coldfirex — COMMENTED** [review]
Sorry, didnt mean to include this in same pull request.

File: content/guides/daily-tasks/netservices.md (hunk 2 above)
Comment (humdingerb) [reply content/guides/daily-tasks/netservices.md]: It actually is "an FTP server". With "a vs. an", it does not depend on how the following word is written (vowel -> an, consonant -> a), but how it's pronounced. "FTP" sounded out is "eff-tee-pee", starting with an "ee" sound, so it's "an FTP server".
Sorry to be such a nitpicker... :)

File: content/get-haiku/installation-guide.html [hunk 4]
```diff
@@ -18,7 +18,7 @@
...
+<p>Currently, Haiku cannot resize existing partitions; therefore, you either need to already have an empty partition where you can install Haiku, or you need to create one using a third party tool. If your current OS comes with a partitioning tool, try to use that. Otherwise, you can use specialized tools like the freely available <a href="http://gparted.sourceforge.net/livecd.php">GParted LiveCD</a>. A standard 32-bit install of Haiku requires about 1.3GiB of disk space (64-bit install is about double), so the partition doesn't need to be huge. However, you may want to leave some room for stuff, so 3GiB+ is the recommended partition size.</p>
```
Comment (humdingerb) [inline content/get-haiku/installation-guide.html:21]: Looks nicer to have a space in front of the GiB.

File: content/guides/daily-tasks/netservices.md (hunk 2 above)
Comment (Coldfirex) [reply content/guides/daily-tasks/netservices.md]: I had no idea, thanks!

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**